pkg/scheduler/framework/session_plugins.go

```diff
@@ -110,9 +125,10 @@ func (ssn *Session) Reclaimable(reclaimer *api.TaskInfo, reclaimees []*api.TaskI
 			if !found {
 				continue
 			}
-			candidates := rf(reclaimer, reclaimees)
+			candidates, unreclaimableList := rf(reclaimer, reclaimees)
```

Comment:
why not candidates, unreclaimableList := rf(reclaimer, candidates)? 
I think this can simplify the following intersection and match.